.byebug_history

```diff
@@ -0,0 +1,23 @@
+c
```

Comment:
Isn't that file supposed to be in .gitignore?

Reply:
It is now; I'll update the branch.